tests/js/client/aql/aql-index-join.js

```diff
@@ -1060,6 +1060,22 @@ const IndexJoinTestSuite = function () {
       }
     },
 
+    testJoinInClusterShardedLocalExpansions: function () {
+      const A = db._create("A", {numberOfShards: 10, replicationFactor: 1, writeConcern: 1, waitForSync: true});
```

Comment:
Do we need the `waitForSync: true` here? I guess it is not needed and will only slow down filling the collection with data.
Same for the following occurrence of `waitForSync: true` below.